nav2_system_tests/worlds/turtlebot3_ros2_demo.world

```diff
@@ -153,6 +153,8 @@
           <plugin name="my_imu_plugin" filename="libgazebo_ros_imu_sensor.so">
             <ros>
```

Comment:
I guess this is a temporal fix. As the plugin knows these are related to `<ros>`, is should deal with passing the args correctly to the node.

Reply:
do you mean a temporary fix? Are you saying these changes shouldn't be needed?

Reply:
I meant it would be preferable that prepending `--ros-args` and `-r` happened upstream
https://github.com/ros-simulation/gazebo_ros_pkgs/issues/991#issuecomment-530160739